tox.ini

```diff
@@ -15,6 +15,7 @@ commands_pre =
 description = run linters
 commands =
     poetry run ruff check
+    poetry run ruff format --check
```

Comment:
@elenbaasc @S-Linde are you OK with this change? Is this a joint decision?

Reply:
I added this line per Stan's suggestion. @S-Linde

Reply:
@S-Linde @rturrado I added this line to the tox.ini, but the pipelines are still failing on the formatting. Do you know what might be the issue in this case?

Reply:
It seems 2 out of 80 files need reformatting. Try running `poetry run ruff check --fix` and `poetry run ruff format` locally.

Reply:
I ran the commands with the changes to tox.ini and it works now :)